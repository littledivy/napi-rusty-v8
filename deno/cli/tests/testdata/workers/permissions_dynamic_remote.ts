new Worker(
  "http://localhost:4545/workers/dynamic_remote.ts",
  {
    type: "module",
    deno: {
      permissions: {
        net: false,
      },
    },
  },
);
