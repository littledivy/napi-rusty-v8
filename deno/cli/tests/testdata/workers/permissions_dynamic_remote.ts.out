error: Uncaught (in worker "") (in promise) TypeError: Requires net access to "example.com", run again with the --allow-net flag
await import("https://example.com/some/file.ts");
^
    at async http://localhost:4545/workers/dynamic_remote.ts:2:1
[WILDCARD]error: Uncaught (in promise) Error: Unhandled error event in child worker.
    at Worker.#pollControl ([WILDCARD])
